src/DssSpell.sol

```diff
@@ -51,91 +45,112 @@ contract DssSpellAction is DssAction {
+        // JAG - 13.02 MKR - 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f
+        // PBG - 0.45 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
 
-        // Decrease the WBTC-C Stability Fee (SF) by 0.07%, from 5.61% to 5.54%
-        DssExecLib.setIlkStabilityFee("WBTC-C", FIVE_PT_FIVE_FOUR_PCT_RATE, /* doDrip = */ true);
+        // Note: payments are skipped on goerli
 
-        // ---------- Reduce PSM-GUSD-A Debt Ceiling ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2
+        // ---------- December Delegate Compensation ----------
+        // Forum: https://forum.makerdao.com/t/december-2023-aligned-delegate-compensation/23352
 
-        // Note: record currently set debt ceiling for PSM-GUSD-A
-        (,,,uint256 lineReduction,) = vat.ilks("PSM-GUSD-A");
+        // 0xDefensor - 41.67 MKR - 0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9
+        // BONAPUBLICA - 41.67 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
+        // Cloaky - 41.67 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
+        // TRUE NAME - 41.67 MKR - 0x612F7924c367575a0Edf21333D96b15F1B345A5d
+        // BLUE - 39.20 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
+        // PBG - 13.89 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
+        // UPMaker - 13.89 MKR - 0xbB819DF169670DC71A16F58F55956FE642cc6BcD
+        // vigilant - 13.89 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
+        // JAG - 12.95 MKR - 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f
+        // WBC - 11.28 MKR - 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47
 
-        // Remove PSM-GUSD-A from `Autoline`
-        DssExecLib.removeIlkFromAutoLine("PSM-GUSD-A");
+        // Note: payments are skipped on goerli
 
-        // Set PSM-GUSD-A debt ceiling to 0
-        DssExecLib.setIlkDebtCeiling("PSM-GUSD-A", 0);
+        // ---------- Offboarded Delegate Buffer Payments ----------
+        // Forum: https://forum.makerdao.com/t/october-2023-aligned-delegate-compensation/22732#october-compensation-2
 
-        // Reduce Global Debt Ceiling? Yes
-        vat.file("Line", vat.Line() - lineReduction);
+        // Navigator - 20.84 MKR - 0x11406a9CC2e37425F15f920F494A51133ac93072
+        // PALC - 6.95 MKR - 0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A
 
-        // ---------- Increase RWA014-A (Coinbase Custody) Debt Ceiling ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2
+        // Note: payments are skipped on goerli
 
-        // Increase the RWA014-A (Coinbase Custody) debt ceiling by 1b DAI, from 500M to 1.5b
-        DssExecLib.increaseIlkDebtCeiling("RWA014-A", 1 * BILLION, /* global = */ true);
+        // ---------- yank Dai streams ----------
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp11-core-unit-offboarding-ses/22332
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp12-core-unit-offboarding-deco/22333
```

Comment:
Currently you have two forum urls together as the justification of the section. Instead, please add those urls on top of each separate action (eg. [example](https://github.com/makerdao/spells-mainnet/blob/44c9b75423d49fabd2dd2c6dc87038dcd7078d04/archive/2023-01-11-DssSpell/DssSpell.sol#L139C1-L150))